tests/controls/dialogtest.cpp

```diff
@@ -132,4 +134,40 @@ void ModalDialogsTestCase::CustomDialog()
     CPPUNIT_ASSERT_EQUAL( 42, dlg.m_value );
 }
 
+
+class MyModalDialog : public wxDialog
+{
+public:
+  MyModalDialog() : wxDialog ( NULL, wxID_ANY, _T("Modal Dialog") )
```

Comment:
`_T()` should not be needed, and neither the spaces before and after the arguments, at least to be consistent with the other constructors.